src/common/composables/use-date-utils/use-date-utils.ts

```diff
@@ -0,0 +1,20 @@
+import type { AvLocale } from '@/types'
+import { format } from 'date-fns'
+import { enUS, fr } from 'date-fns/locale'
+import { useI18n } from 'vue-i18n'
+
+export function useDateUtils () {
```

Comment:
```suggestion
/**
 * Result returned by the useDateUtils composable.
 */
interface UseDateUtilsReturn {
  /**
   * Provide the purpose of the method
   *
   * @example
   * ```ts
   * // provide an example
   * ```
   */
  formatTranslatedDateTime: (date: string) => string
}

/**
 * Vue composable to handle .... (a short description in introduction).
 *
 * This composable provides .... (a longuer description in body).
 *
 * @example
 * ```ts
 * // Provide an example of use
 * ```
 *
 * @returns {UseDateUtilsReturn} Object containing:
 *  - ....
 */
export function useDateUtils (): UseDateUtilsReturn {
```

Il faut qu'on prenne le réflexe de documenter nos composables :+1: